app/src/main/res/layout/fragment_auth.xml

```diff
@@ -4,13 +4,69 @@
     android:layout_height="match_parent"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <com.google.android.gms.common.SignInButton
-        android:id="@+id/btn_google_sign_in"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+    <ImageView
+        android:id="@+id/iv_plant_image"
+        android:layout_width="0dp"
+        android:layout_height="@dimen/auth_image_height"
+        android:scaleType="fitXY"
+        android:src="@drawable/auth_image"
         app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent" />
+        app:layout_constraintEnd_toEndOf="parent"/>
+
+    <TextView
+        android:id="@+id/tv_app_name"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_32dp"
+        android:fontFamily="@font/roboto_bold"
+        android:text="@string/app_name"
+        android:textAlignment="center"
+        android:textColor="@color/green_primary"
+        android:textSize="32sp"
```

Comment:
you can move sizes of text to dimens too
like 
text_size_huge or something like this